entity-framework/core/testing/index.md

```diff
@@ -1,117 +1,116 @@
 ---
-title: Testing code that uses EF Core - EF Core
+title: Testing applications that use EF Core - EF Core
 description: Different approaches to testing applications that use Entity Framework Core
-author: ajcvickers
-ms.date: 04/22/2020
+author: roji
+ms.date: 11/07/2021
 uid: core/testing/index
 ---
-# Testing code that uses EF Core
+# Testing EF Core Applications
 
-Testing code that accesses a database requires either:
+Testing is an important concern to almost all application types - testing allows you to be sure your application work correctly, and makes it instantly known if its behavior regresses in the future. Since testing may affect how your code is architected, it's highly recommended to plan for a testing early, and to ensure good coverage as your application evolves. This section provides an overview of various testing strategies for applications using EF Core, shows how each one can be implemented and discusses best practices.
```

Comment:
Grammar: "your application work correctly"

Reply:
Also, Oxford comma missing.

Reply:
This is a very opinionated statement about testing applications. Are we really the ones who should be saying. "you must test!" as opposed to saying, "if you want to test, this might help?"

Reply:
I agree it's "programmatic", though I wouldn't say this is very controversial :) I think it's OK as a general introductory blurb, but if you believe this is too much I can tone it down.

Reply:
I asked a scholar from Oxford, and they said the comma must be used. I then asked a scholar from Cambridge, and they agreed. If scholars from Oxford and Cambridge agree on something, then it must be right.